### ISSUE TITLE
Update title for chapter "using SVN"

### ISSUE THREAD

**Issue by closier** [lhcb/first-analysis-steps#191]
The Chapter "using SVN" could probably rename "Using SVN for LHCb Doc". It is specific to write LHCb documentation for analysis and not anymore for CODE. What do you think ?

**Comment by alexpearce** [comment]
I agree. I think we need a general overhaul to convert everything to git and `lb-run` for code development and `GaudiExec` for running things in Ganga, see #188 for the latter.

(Having said that, there are still a few projects that are stuck on SVN for now, such as `DecFiles`, which are likely to be touched by analysts at some point.)
